How-To-Load-Data.md

```diff
@@ -0,0 +1,18 @@
+### Populating translation opportunities
+- Start a local server and sign in as an administrator (testadmin@example.com, assume username “a”).
+- Navigate to the Admin page.
+- Click the Roles tab.
+- In the Role editor, enter the admin username (“a”). Click Add Role, and add the “Curriculum Admin” role.
+- Navigate to the Activities tab. Click Load Data under Load dummy new structures data.
+- You should now see translation opportunities in the Translate Text tab at /contributor-dashboard.
```

Comment:
If these are steps convert them to a numbered list.

Reply:
Done